plugin/customcertificate/src/export_pdf_all_in_one.php

```diff
@@ -0,0 +1,661 @@
+<?php
```

Comment:
Codacy found an issue: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5744985)